### ISSUE TITLE
The chinese name of RChain 

### ISSUE THREAD

**Issue by sportshark** [rchain/Members#174]
What is the chinese name of RChain?

**Comment by sportshark** [comment]
I find a name "微积链"
微积 is calculus in chinese
链 is chain in chinese

**Comment by Mervyn853** [comment]
Seems good, I know some wanted a standard RChain brand across all languages but even bitcoin (比特幣) and ether (以太幣) have chinese translations come up. I find 微积链 fitting and apt.

**Comment by lapin7** [comment]
That’s a nice name 微积链 
Google translates it as 
Calculus chain

You’re right it needs à Chinese name

Best,
HJ

**Comment by lapin7** [comment]
Otherwise it would be 

R 链

And R doesn’t exist in Chinese?

Best,
HJ

**Comment by sportshark** [comment]
R is a English letter in Chinese.In 2016 when RChain is a part of Synereo,It is called R链。

**Comment by sportshark** [comment]
We can collect some names and choose one from them

**Comment by Mervyn853** [comment]
Hi Andy, who is 毛 in QQ chat? Can you check your telegram rchain Chinese group? Have a few issues to discuss with the name

**Comment by sportshark** [comment]
In Chinese  QQ group  ,Echo suggests that the RChain name is “瑞链”，瑞 represents luck and happiness.
And some Chinese  people think R链 is  a good name.

**Comment by lapin7** [comment]
What has become of this naming?
I think the R + something of 微积链 would become a great name.

**Comment by pmoorman** [comment]
@Mervyn853 @sportshark what has become of this discussion? Do we have a conclusion here?

I'll label this issue "branding" rather than "marketing" btw. If someone could post the conclusion of the discussion, that would be nice.

**Comment by cool007zqw** [comment]
微积 sounds exactly the same as 危机（crisis）in Mandarin so I would oppose it
In fact in China most people will call RChain RChain or 日链 (日 = sun as a noun in Chinese, but can be interpreted as 'fuck' if one uses it as a verb)
IMO how people will call somthing is not decided by a certain group so how about let's just wait and see how things will go.

**Comment by allancto** [comment]
@Shakakai @ottermagically @zsluedem i agree with the sentiment of going with experienced people on decisions like this, with one caveat. The names RChain, rholang and so on and so on were conceived out of (dare i say?) love and not by professionals. So there are many paths that can lead to a good way. How to recognize what's cool and what's drool, seems to be some kind of art. -@allancto

**Comment by Viraculous** [comment]
@zluedem your comment above indicated that you hope for more discussions on this, is there any progress so far on it? Did you guys come up with something?

**Comment by zsluedem** [comment]
@Viraculous No progress at all these days.
